apps/tsilhqotin-language-hub/src/Components/Pages/Funders/Funders.tsx

```diff
@@ -1,4 +1,4 @@
-import { Divider } from '@mui/material';
```

Comment:
We should definitely pull funders from a DB at some point. For now could you have some data locally

```ts
   type FunderViewModel = {
   name: string;
   summary: string;
   priority: number; // or Year or something to determine order
   isCurrentFunder: boolean;
}

  const funderData: FunderViewModel[] = [...]; 
```